### PR TITLE
No need to wait after Wire.requestFrom()

### PR DESCRIPTION
**Pull request by Koepel**

### CODE REVIEW

File: SHTSensor.cpp [hunk 1]
```diff
@@ -73,14 +73,9 @@ bool SHTI2cSensor::readFromI2c(uint8_t i2cAddress,
 
   Wire.requestFrom(i2cAddress, dataLength);
 
-  // there should be no reason for this to not be ready, since we're using clock
-  // stretching mode, but just in case we'll try a few times
-  uint8_t tries = 1;
-  while (Wire.available() < dataLength) {
-    delay(1);
-    if (tries++ >= MAX_I2C_READ_TRIES) {
```
Comment (winkj) [inline SHTSensor.cpp:81]: Looks good overall, but MAX_I2C_READ_TRIES should be removed, too, since it's not needed anymore

**Review by winkj — CHANGES_REQUESTED** [review]

File: SHTSensor.cpp (hunk 1 above)
Comment (Koepel) [reply SHTSensor.cpp:81]: Yes, the MAX_I2C_READ_TRIES can be removed as well.

**Comment by winkj** [comment]
Please also remove MAX_I2C_READ_TRIES in SHTSensor.h, and combine the three commits into one since they are one logical change

**Comment by Koepel** [comment]
I'm sorry, but I can't find a Merge button for an existing Pull Request. When I make a new Pull Request I'm redirected to this existing Pull Request.

**Comment by winkj** [comment]
Your master repository still has three commits that should be squashed into one; you can either do that with "git rebase", or with a temporary branch. Let me know if you need any help with that!

**Comment by winkj** [comment]
@Koepel actually, I can squash them during the merge process; I'd still recommend to bundle these changes in the future, but for this request, we can do that.

@abrauchli comments?

**Comment by abrauchli** [comment]
@Koepel lgtm, thanks, will squash-merge.

**Comment by abrauchli** [comment]
Merged as d999cbce3d604e6118d84e119f6cc3a94c5db488

**Pull request closed without merging**